features/bevragen/zoek-met-nummeraanduiding-identificatie/dev/parameter-autorisatie-gba.feature

```diff
@@ -0,0 +1,88 @@
+      En de persoon met burgerservicenummer '000000024' heeft de volgende 'verblijfplaats' gegevens
+      | gemeente van inschrijving (09.10) |
+      | 0599                              |
+      En de 'verblijfplaats' heeft de volgende 'adres' gegevens
+      | gemeente_code | identificatiecode nummeraanduiding (11.90) |
+      | 0599          | 0599200000219679                           |
+      Als gba personen wordt gezocht met de volgende parameters
+      | naam                          | waarde                               |
+      | type                          | ZoekMetNummeraanduidingIdentificatie |
+      | nummeraanduidingIdentificatie | 0599200000219679                     |
+      | fields                        | burgerservicenummer                  |
+      Dan heeft de response een object met de volgende gegevens
+      | naam     | waarde                                                                                      |
+      | type     | https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.3                                 |
+      | title    | U bent niet geautoriseerd voor de gebruikte parameter(s).                                   |
+      | status   | 403                                                                                         |
+      | detail   | U bent niet geautoriseerd voor het gebruik van parameter(s): nummeraanduidingIdentificatie. |
+      | code     | unauthorizedParameter                                                                       |
+      | instance | /haalcentraal/api/brp/personen                                                              |
+
+    @fout-case
+    Scenario: Gebruik van parameter gemeenteVanInschrijving waarvoor de afnemer niet geautoriseerd is (80910)
+      Gegeven de afnemer met indicatie '12345' heeft de volgende 'autorisatie' gegevens
+      | Rubrieknummer ad hoc (35.95.60)                       | Medium ad hoc (35.95.67) | Datum ingang (35.99.98) |
+      | 10120 81010 81110 81120 81130 81140 81150 81160 81190 | N                        | 20201128                |
+      En de geauthenticeerde consumer heeft de volgende 'claim' gegevens
+      | naam         | waarde |
+      | afnemerID    | 12345  |
+      | gemeenteCode | 0518   |
+      En de persoon met burgerservicenummer '000000024' heeft de volgende 'verblijfplaats' gegevens
+      | gemeente van inschrijving (09.10) |
+      | 0599                              |
+      En de 'verblijfplaats' heeft de volgende 'adres' gegevens
+      | gemeente_code | identificatiecode nummeraanduiding (11.90) |
+      | 0599          | 0599200000219679                           |
+      Als gba personen wordt gezocht met de volgende parameters
+      | naam                          | waarde                               |
+      | type                          | ZoekMetNummeraanduidingIdentificatie |
+      | nummeraanduidingIdentificatie | 0599200000219679                     |
```

Comment:
Voer dit veld en waarde op in database. Maakt het scenario realistischer.

Reply:
ik snap deze opmerking niet. Ik heb toch bij de Gegeven stappen opgevoerd:
```
En de persoon met burgerservicenummer '000000024' heeft de volgende 'verblijfplaats' gegevens
      | gemeente van inschrijving (09.10) |
      | 0599                              |
      En de 'verblijfplaats' heeft de volgende 'adres' gegevens
      | gemeentecode (92.10) | identificatiecode nummeraanduiding (11.90) |
      | 0599                 | 0599200000219679                           |
```